setup.cfg

```
@@ -0,0 +1,57 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for
+ the microservice on aiohttp and prepare the development
+ environment.
+long_description = file: README.md
+long_description_content_type = 'text/markdown'
+platforms = 
+    POSIX
+author = Mykhailo Havelia
+author_email = misha.gavela@gmail.com
+license = MIT
+url = https://github.com/aio-libs/create-aio-app
+project_urls = 
+    Source Code = https://github.com/aio-libs/create-aio-app
+    GitHub Issues = https://github.com/aio-libs/create-aio-app/issues
+    Documentation = https://create-aio-app.readthedocs.io/
+    CI: Travis = https://travis-ci.com/aio-libs/create-aio-app
+
+classifiers =
+    License :: OSI Approved :: MIT License
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
```

Let's add Python 3.8 too:
```suggestion
    Programming Language :: Python :: 3.8
```